src/flag.rs

```diff
@@ -122,7 +129,7 @@ pub struct MigrationFlagParameters {
     /// read or write operation. This value can be controlled through the LaunchDarkly UI and
     /// propagated downstream to the SDKs.
     #[serde(skip_serializing_if = "is_default_ratio")]
-    pub check_ratio: Option<u64>,
+    pub check_ratio: Option<u32>,
```

Comment:
Is the u64->u32 just because we know that the JSON parser won't be handling 64 bit ints?